examples/caps-miller.jl

```diff
@@ -0,0 +1,41 @@
+# ~/~ begin <<docs/src/ca-with-production.md#examples/caps-osc.jl>>[init]
+using CarboKitten
+using CarboKitten.CaProd
+using CarboKitten.Burgess2013
+using CSV
+using DataFrames
+using Interpolations
+
+
+function sealevel_curve(t,filepath)
+    data = DataFrame(CSV.File(filepath))
+    data = hcat(collect(0:length(data[:,1]).-1)./1000, data[:,1]) #The output sealevel curve from R does not have time tab and have to add it 
+    x = linear_interpolation(data[:,1], data[:,2])
+    return x(t)
+end 
+
+
+DEFAULT_INPUT = CaProd.Input(
+    sea_level = t -> sealevel_curve(t,"data/miller.csv"),
```

Comment:
This file, which is [cited as an example in the documentation](https://github.com/MindTheGap-ERC/CarboKitten.jl/blob/b1a8803c6e856988716e7d8eafe774a9f28fa82e/docs/src/Denudation.md?plain=1#L180), cannot work, because `data/miller.csv` is missing.